utils.js

```diff
@@ -0,0 +1,155 @@
+var setDefaultCity = function() {
+  $('#default-city').empty();
+  var defaultCity = 'Default City: <b>' + localStorage.getItem('default') + '</b>';
+  $('#default-city').append(defaultCity);
+}
+
+//posts current weather information for inputted city
+var postCurrent = function (data) {
+  $('#weather-info').empty();
+  $('#weather-info-img').empty();
+
+  let temp = Math.round(data.main.temp);
+
+  var weatherInfo = '<div class="temp">' + temp + '°</div><div class=city>' + data.name + '</div><div>' + data.weather[0].main + '</div>';
+  $('#weather-info').append(weatherInfo);
+
+  var defaultButtonSet = function(data) {
```

Comment:
This seems a little fragile. Using a template would make this easier to read and maintain